packages/edit-widgets/src/components/customizer-layout/style.scss

```diff
@@ -0,0 +1,22 @@
+.edit-widgets-customizer-layout__content {
+	background: #f1f1f1;
+	margin: 0;
+	min-height: 100%;
+	padding: 30px 0;
+
+	.block-editor {
+		&-block-list {
```

Comment:
I find this easier to read. But I didn't think that it could break search. I think that alone is enough reason to not nest it.

Editors should have a setting for parsing nested selectors in their search...